playwright-tests/properties-page-tests.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { test, expect } from '@playwright/test';
+import { properties } from './helpers';
+
+
+// This test uses 2 properties
+const firstProperty = properties[0];  // 1 Test Road
+const secondProperty = properties[1]; // 2 Test Road
+
+test('Existing property test: Address', async ({ page }) => {
+  await page.goto(`http://localhost:3000/properties/${firstProperty.id}`);
+  await expect(page.getByRole('main')).toContainText('1 Test Road');
```

Comment:
You added the property address to your helper object, but aren't using it here.

If you are going to do it this way, I would expect this line to read
`await expect(page.getByRole('main')).toContainText(firstProperty.address);`